src/send_intro.py

```diff
@@ -0,0 +1,165 @@
+                colour=disnake.Colour(2357273),
+            ).set_thumbnail(
+                url="https://pngimg.com/uploads/scales/scales_PNG10.png"
+            ),
+
+          disnake.Embed(
+              title="Membership",
+              description="*FU membership means identifying with the goals and values of the community. \nBecoming a member is **your choice**, not something we need you to become.\nYou can still play with us without being a member.*\n\nTo become a member:\n:white_small_square: (Optional but recommended) Read the [Introduction](https://docs.google.com/document/d/1TRdV8-9JxXg9gLIp_PC56T5GyTiPxT1uxVLM1_9xRu4/edit?usp=sharing) document\n:white_small_square: (1/2) Contact an Officer about an introduction meeting on TS\n:white_small_square: (2/2) Attend a scheduled introduction meeting. See [#schedule](https://discordapp.com/channels/282514718445273089/539192935258783744)\n:white_small_square: During the meeting the Officer will discuss any questions you have regarding FU (assuming you've read the intro document or played with us for some time)\n:white_small_square: You will be offered FU membership during the meeting which you can accept or reject. \n:white_small_square: If you reject membership you will be given Guest status on our discord and not notified again about introduction events. You may at any time ask for membership should you change your mind.\n\nMembership gives you the member rank in PlanetSide, the **[FU]** tag on TS and exclusive access to the Discord member's channel category. \n\n**Membership is completely optional**, accept if you wish to associate with the community and wear the **[FU]** tag.",
+              colour=disnake.Colour(2357273),
+          ).set_thumbnail(
+            url="https://i.pinimg.com/originals/1a/a0/47/1aa04756777580a5328ad9f456c9dadb.png"
+          ),
+        ]
+        return embeds
+
+    @commands.Cog.listener()
+    async def on_member_join(self, member):
+
+        # Send a DM to the new member
+        embeds = await self.paginator()
+        await member.send(content=None, embed=embeds[0], view = Menu(embeds))
+
+    @commands.slash_command(dm_permission=True)
+    async def intro(self, inter):
+        """
+        Send a DM of the intro message
+        """
+        await inter.response.send_message("Sending intro message to DM...", ephemeral=True)
+        embeds = await self.paginator()
+        await inter.author.send(content=None, embed=embeds[0], view = Menu(embeds))
```

Comment:
```suggestion
        await inter.author.send(content=None, embed=embeds[0], view=Menu(embeds))
```
Aha! I found something!